lib/cmd/fh3/call.js

```diff
@@ -0,0 +1,49 @@
+/* globals i18n */
+var fhreq = require("../../utils/request");
+
+module.exports = {
+  'desc' : i18n._('Call a Red Hat Mobile Application Platform system URL'),
```

Comment:
My pick 
"Command used to call any api endpoint as currently authenticated user"